README.md

```diff
@@ -0,0 +1,76 @@
+# Gitlab-Slack Notifier
+
+This project has the objective to integrate Gitlab with Slack teams through webhooks. We create a service in NodeJS, that handle the hooks and send messages to channels and direct messages to slack users. 
+
+So far, we integrated *Push Hooks* and *Merge Request Hooks*.
+
+**IMPORTANT:** If you want to notify the slack users through DM, they must have the same username in Gitlab and Slack.
+##### Ex: Slack username: @brunojdo and Gitlab username: brunojdo
+
+### Specs
+
+* NodeJS 8.3
+* Express 4.10.*
+
+## Usage
+
+### Docker
+
+You can start the service using container. Execute the command below: 
+
+`docker run -d --restart=always -p 8008:8080 --name notifier-slgb brunodias20/slack-gitlab-notifier:{version}`
+
+### NPM 
+
+If you prefer, you also may clone this project, and into `app/` folder, execute: 
+
+`npm run build && npm run serve`
+
+## Setup
+
+### Enable Gitlab Webhooks
+
+You need to enable Webhooks in your gitlab. Go into your project **Settings -> Integrations** and put your service address. For example: `http://localhost:8008/webhook` and select which hooks you want use. For more informations [click here.](https://docs.gitlab.com/ce/user/project/integrations/webhooks.html)
```

Comment:
For example: `http://localhost:8008/webhook` and select which hooks you want use

For example, on: `http://localhost:8008/webhook` select